src/sum_from_string.py

```diff
@@ -0,0 +1,16 @@
+"""A function to sum numbers found in a string."""
+
+
+def sum_from_string(string):
+    """Sum numbers found in a string."""
+    new_l = []
+    for i in list(string):
+        try:
+            int(i)
```

Comment:
This line will not work. `int` returns a new value, and you aren't doing anthing with that new value.